avs-aggregator/core/chainio/avs_writer.go

```diff
@@ -97,3 +103,24 @@ func (w *AvsWriter) SendAggregatedResponse(ctx context.Context, task taskmanager
 	w.logger.Info("sent aggregated response with the AVS's task manager")
 	return receipt, nil
 }
+
+func (w *AvsWriter) EjectOperators(ctx context.Context, operators []common.Address, quorumNumbers[][]uint8) (*types.Receipt, error) {
+	w.logger.Info("sending eject operators with AVS's service manager")
+	noSendTxOpts, err := w.txMgr.GetNoSendTxOpts()
+	if err != nil {
+		return nil, err
```

Comment:
missing log here?

Reply:
the logs are in the upper level, where the kicker calls the eject method